common/lib/xmodule/xmodule/html_module.py

```diff
@@ -459,10 +463,9 @@ def get_html(self):
 
         # When we switch this to an XBlock, we can merge this with student_view,
         # but for now the XModule mixin requires that this method be defined.
-        if self.data != "":
-            if self.system.anonymous_student_id:
-                return self.data.replace("%%USER_ID%%", self.system.anonymous_student_id)
-            return self.data
+        data = HtmlBlock.get_html(self)
```

Comment:
Does `super().get_html()` not work here?

Reply:
It does, good catch!